.gitignore

```diff
@@ -34,6 +34,7 @@ yarn.lock
 *.launch
 .settings/
 *.sublime-workspace
+.vscode
```

Comment:
Merge line 37 with line 40 into a single regular expression.

Reply:
I actually didn't notice the existing vscode section before 🤦🏻‍♂️. Removed this line entirely.